signatures/reservations.yml

```diff
@@ -5,22 +5,25 @@ Reservation:
     title: Create Reservation
     description: Create Reservation instance, also record the creation time
     params:
-      disease_id:
+      disease:
         type: Number
         description: Mandatory disease ID.
-      hospital_id:
+      hospital:
         type: UUID
         description: Mandatory hospital ID.
-      patient_id:
+      patient:
         type: UUID
         description: Mandatory patient ID.
-      timeslot_id:
+      timeslot:
         type: UUID
         description: Mandatory reservation slot ID.
-      user_id:
+      customer:
         type: UUID
         description: Mandatory customer ID.
-    returns: {}
+    returns:
+      rid:
+        type: String
```

Comment:
This seem to be a UUID https://gitlab.com/yiconfirmed/backend/blob/develop/reservation/models.py#L16